be/src/exec/pipeline/scan/connector_scan_operator.cpp

```diff
@@ -38,7 +38,9 @@ OperatorPtr ConnectorScanOperatorFactory::do_create(int32_t dop, int32_t driver_
 
 ConnectorScanOperator::ConnectorScanOperator(OperatorFactory* factory, int32_t id, int32_t driver_sequence,
                                              ScanNode* scan_node)
-        : ScanOperator(factory, id, driver_sequence, scan_node) {}
+        : ScanOperator(factory, id, driver_sequence, scan_node) {
+    set_io_tasks_per_op(config::connector_io_tasks_per_scan_operator);
+}
```

Comment:
How about making `io_tasks_per_scan_operator` as the argument of the constructor of `ScanOperator`, and pass different values in the constructor of `ConnectorScanOperator` and `OlapScanOperator`?

Reply:
OK.